src/coreclr/vm/appdomain.cpp

```diff
@@ -1041,6 +1041,7 @@ void SystemDomain::Attach()
 
     // Create the one and only app domain
     AppDomain::Create();
+    AppDomain::GetCurrentDomain()->CreateBinderContext();
```

Comment:
I wonder if this can be moved inside the `Create` call above. Probably inside Init if we can create binding context with "half-initialized" appdomain.

Reply:
I think we could move this into `Create`. `CreateBinderContext` does not do much besides allocating the binder and raising ETW event. 
Maybe should be renamed `CreateDefaultBinder` as well. I will follow up in next PR.